pages/linux/synoupgrade.md

```diff
@@ -0,0 +1,36 @@
+# synoupgrade
+
+> Upgrade a Synology DiskStation Manager (DSM) from the command-line.
+> More information: <https://www.synology.com/dsm>.
+
+- Check upgrades printing the result:
```

Comment:
I don't think the `printing the results` part is necessary. I'd even rephrase it a bit:
```suggestion
- Chech if there are upgrades available:
```
or
```suggestion
- Check if upgrades are available:
```